app/src/app/page.tsx

```diff
@@ -1,17 +1,113 @@
+              >
+                <div className="avatar mask mask-squircle w-12 h-12">
+                  <img alt={prato.nome} src="https://cdn.tasteatlas.com/images/dishes/9c4888bb938346c3ada2cddd5d1a0ebc.jpg" />
+                </div>
+                <div>
+                  <div className="font-bold">{prato.nome}</div>
+                </div>
+              </div>
+            ))}
+          </td>
+          <td>
+            {order.bebidas.map((bebida) => (
+              <div
+                key={bebida.id}
+                className="mb-1 flex items-center gap-3 bg-gray-400 p-4 rounded-lg"
+              >
+                <div className="avatar mask mask-squircle w-12 h-12">
+                  <img alt={bebida.nome} src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ1FAP0Qz-w0QwFTUppkmJJ-Ovd4xLKkZaVNv9r6gouvA&s" />
+                </div>
+                <div>
+                  <div className="font-bold">{bebida.nome}</div>
+                </div>
+              </div>
+            ))}
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  ));
+
+  const pendingOrdersTables = pendingOrders.map((order) => (
+    <table
+      key={order.id}
+      className="table-lg w-full bg-gray-300 mb-4 rounded-lg bg-opacity-70"
+    >
+      <thead>
+        <tr>
+          <th className="text-left w-1/2">Order {order.id}</th>
+          <th className="text-right w-1/2">Table {order.mesa}</th>
+        </tr>
+      </thead>
+      <tbody className="align-top">
+        <tr>
+          <td>
+            <div className="mb-1 flex items-center gap-3 bg-gray-400 p-4 rounded-lg">
+              <div>
+                <div className="font-bold">{order.pratos.length} Pratos</div>
+              </div>
+            </div>
+          </td>
+          <td>
+            <div className="mb-1 flex items-center gap-3 bg-gray-400 p-4 rounded-lg">
+              <div>
+                <div className="font-bold">{order.bebidas.length} Bebidas</div>
+              </div>
+            </div>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  ));
 
-export default function Home() {
   return (
-    <main className="flex min-h-screen flex-col items-center justify-between p-24">
-      <div className="card w-96 bg-base-100 shadow-xl">
-        <figure><img src="https://img.daisyui.com/images/stock/photo-1606107557195-0e29a4b5b4aa.jpg" alt="Shoes" /></figure>
-        <div className="card-body">
-          <h2 className="card-title">Shoes!</h2>
-          <p>If a dog chews shoes whose shoes does he choose?</p>
-          <div className="card-actions justify-end">
-            <button className="btn btn-primary">Buy Now</button>
-          </div>
-        </div>
+    <main className="m-4 flex">
+      <div className="w-2/3 mr-4 bg-green-400 p-4 rounded-lg">
+        <h2 className="text-2xl font-bold mb-4">Orders In Progress</h2>
+        {inProgressOrdersTables}
+      </div>
```

Comment:
Dividing this with `2 divs` inside that `div` with fixed size could be nice (one for the h2 and the other one for the table). @fungame2270 give feedback pls.

Reply:
true